homeassistant/helpers/deprecation.py

```diff
@@ -194,10 +213,56 @@ def _print_deprecation_warning(
         else:
             logger.warning(
                 "%s was %s from %s, this is a deprecated %s%s. Use %s instead",
-                obj.__name__,
+                obj_name,
                 verb,
                 integration_frame.integration,
                 description,
                 breaks_in,
                 replacement,
             )
+
+
+DeprecatedConstant = namedtuple(
```

Comment:
Can we use new style `NamedTuple` instead?

https://docs.python.org/3/library/typing.html#typing.NamedTuple